Class2/Program.cs

```diff
@@ -4,19 +4,38 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            //string,bool, int, float, list, dic,char
-            var list = new List<string>() { "krishna","radha"};
-            var dic = new Dictionary<string, int>();
-            dic["age"] = 20;
-            //Console.Write("x:");
-            //var x = Console.ReadLine();
-            //Console.Write("y:");
-            //var y =Console.ReadLine();
-            //var z = x.ToString();
-            var name3 = $"{dic["Husband"]} and {dic["Wife"]} are husband and wife";
-            Console.WriteLine(name3);
-            Console.WriteLine("Hello, everyone!");
-            Console.WriteLine("Hello, Class!");
+            Console.WriteLine("This program lets you enter 6 numbers and \n" +
+                "provides you with pair of numbers that is equal to the sum of your choice.\n");
+            var nums = new List<int>();
+            int numSum;
+            bool pairs = false;
+
+            for (int l = 0; l < 6; l++)
+            {
+                Console.WriteLine("Please enter number" + (l + 1) + " and press enter:");
```

Comment:
(1+1) ko reason?

Reply:
Lol, tyo L+1 ho sir. number1, Number 2.. number6 display garna. So that user knows how many numbers have they entered.